app/scripts/vendor/env-test.js

```diff
@@ -81,4 +81,11 @@
   if (isStyleAllowed(style)) {
     docElement.className += (' ' + style);
   }
+
+  /**
+   * Check if the user is signing in to Sync on Firefox for iOS. User agent sniff. Yuck.
+   */
+  if (/FxiOS/.test(navigator.userAgent) && parseQueryParams().service === 'sync') {
```

Comment:
> Not sure if we can use the isSync() helper here /app/scripts/models/reliers/relier.js:70

Normally I'd say YES! Not here though, vender/env-test.js loads up in the HEAD of the document so that we can attach styles before the page is rendered. We try to make it as small as possible to avoid delaying page render any longer than necessary.

Unrelated; this module has no unit tests and that makes me uneasy.